### DISCUSSION TITLE
No Download option available?

### DISCUSSION THREAD

**Discussion by forsyth47** [imashnake0/Animite#21] [General]
Um sorry but, is this app still in development? Cuz it looks really cool from the mock-up and I wanna try it out!

**Comment by imashnake0** [comment]
Glad you like it! It's still in development, I'll create a milestone board soon to reflect future plans and progress.

**Comment by imashnake0** [comment]
@forsyth47 I started uploading APKs to GitHub, feel free to try it out: https://github.com/imashnake0/Animite/releases/tag/v0.0.1-alpha01. Note that it's not perfect yet.
